core/templates/dev/head/forms/formBuilder.js

```diff
@@ -559,6 +561,9 @@ oppia.factory('rteHelperService', [
           RTE_COMPONENT_SPECS[componentId].customization_arg_specs),
         name: RTE_COMPONENT_SPECS[componentId].frontend_name,
         iconDataUrl: RTE_COMPONENT_SPECS[componentId].icon_data_url,
```

Comment:
Ah, sorry, my mistake. Yep, let's have two things for each component: a toolbar-icon and a preview-image.